data-processed/KITmetricslab-bivar_branching/metadata-KITmetricslab-bivar_branching.txt

```diff
@@ -1,21 +1,19 @@
 team_name: KITmetricslab
-model_name: bivariate_branching
```

Comment:
You can actually keep this line as it was if you wish. The most important is to edit the line below: `model_abbr`. It is the one with the character limit